docs/usage.md

```diff
@@ -99,114 +92,56 @@ vds.vz.nbytes
-
-The default value of `loadable_variables` is `None`, which effectively specifies all the "dimension coordinates" in the file, i.e. all one-dimensional coordinate variables whose name is the same as the name of their dimensions. Xarray indexes will also be automatically created for these variables. Together these defaults mean that your virtual dataset will be opened with the same indexes as it would have been if it had been opened with just `xarray.open_dataset()`.
+The default value of `loadable_variables` is `None`, which effectively specifies all the "dimension coordinates" in the file, i.e. all one-dimensional coordinate variables whose name is the same as the name of their dimensions. Xarray indexes will also be automatically created for these variables. Together these defaults mean that your virtual dataset will be opened with the same indexes as it would have been if it had been opened with just [xarray.open_dataset][].
 
 !!! note
     In general, it is recommended to load all of your low-dimensional (e.g scalar and 1D) variables.
 
-    Whilst this does mean the original data will be duplicated in your new virtual zarr store, by loading your coordinates into memory they can be inlined in the reference file, or be stored as single chunks rather than large numbers of extremely tiny chunks. (Both of which will speed up loading that data back when you re-open the virtual store later.)
+    Whilst this does mean the original data will be duplicated in your new virtual zarr store, by loading your coordinates into memory they can be inlined in the reference file or be stored as single chunks rather than large numbers of extremely tiny chunks, which speeds up loading that data on subsequent usages of the virtual dataset.
 
     However, you should not do this for much higher-dimensional variables, as then you might use a lot of storage duplicating them, defeating the point of the virtual zarr approach.
 
     Also, anything duplicated could become out of sync with the referenced original files, especially if not using a transactional storage engine such as `Icechunk`.
 
 ### Loading CF-encoded time variables
 
-To decode time variables according to the CF conventions upon loading, you must ensure that variable is one of the `loadable_variables` and the `decode_times` argument of `open_virtual_dataset` is set to `True` (`decode_times` defaults to None).
+To decode time variables according to the CF conventions upon loading, you must ensure that variable is one of the `loadable_variables` and the `decode_times` argument of `open_virtual_dataset` is set to `True` (`decode_times` defaults to `None`).
 
-```python
+```python exec="on" session="usage" source="material-block" result="code"
 vds = open_virtual_dataset(
-    'air.nc',
-    object_store=LocalStore,
-    parser=HDFParser(),
-    loadable_variables=['air', 'time'],
+    url=url,
+    registry=registry,
+    parser=parser,
+    loadable_variables=['time'],
```

Comment:
@TomNicholas do you think we still need the section on "Loading CF-encoded time variables", now that we use the default loading of xarray (which includes 1D time coordinates) and open_zarr (which defaults to `decode_times=True`)

Reply:
It's far less likely to come up now, but technically we do still need this. Someone could have a variable that is not one that will be loaded by default (because it's a data variable, or a >1D coordinate variable, or it is a 1D coordinate variable that has a different name from its dimension).